airbyte-integrations/connectors/source-yandex-metrica/source_yandex_metrica/streams.py

```diff
@@ -0,0 +1,345 @@
+
+class Sessions(IncrementalYandexMetricaStream):
+    def __init__(self, counter_id: str, params: dict, **kwargs):
+        self.fields_list = params['fields']
+        fields = ','.join(params['fields'])
+        params['source'] = YMSource.SESSIONS
+        params['fields'] = fields
+        super().__init__(counter_id, params, **kwargs)
+
+    def path(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return f"{self.counter_id}/logrequests/evaluate"
+    
+    def get_json_schema(self) -> Mapping[str, any]:
+        schema = super().get_json_schema()
+        schema = {
+            '$schema': 'http://json-schema.org/draft-04/schema#', 
+            'type': 'object', 
+            'properties': {}
+        }
+        fields = {key: VisitsFields.get_all_fields()[key] for key in self.fields_list}
+        for key, value in fields.items():
+            schema['properties'][key] = value
+
+        return schema
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        return super().request_params(stream_state=stream_state)
+
+    @property
+    def raise_on_http_errors(self) -> bool:
+        return False
+
+    def parse_response(
+        self,
+        response: requests.Response,
+        stream_state: Mapping[str, Any] = None,
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping]:
+        yield from self.fetch_records(response, stream_state)
+
+# Substreams
+class Create(YandexMetricaStream):
+    primary_key = None
+
+    def __init__(self, counter_id: str, params: dict, **kwargs):
+        super().__init__(counter_id, params, **kwargs)
+
+    @property
+    def http_method(self) -> str:
+        return "POST"    
+
+    def path(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return f"{self.counter_id}/logrequests"
+    
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+
+        return {
+            'date1': self.params['start_date'],
+            'date2': self.params['end_date'],
+            'source': self.params['source'],
+            'fields': self.params['fields']
+        }
+
+    def parse_response(
+        self,
+        response: requests.Response,
+        stream_state: Mapping[str, Any] = None,
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping]:
+        data = response.json()
+        yield data
+class Check(YandexMetricaStream):
+    primary_key = None
+
```

Comment:
Please run `./gradlew --no-daemon  :airbyte-integrations:connectors:<connector_name>:airbytePythonFormat`

Reply:
What is this for?

Reply:
@ernestasga merge with latest master and run again

Reply:
> @ernestasga merge with latest master and run again

Thanks. This helped. Can you also check my other comment about failing normalization? I'm stuck.